### PR TITLE
remove signers library

### LINKED ISSUE #771: Remove usage of signers library
Remove the usage of the signers library and inline the code as a module in web3signer. This removes additional overhead from the development and release process. 

Move the following modules
- [x] keystore to web3signer
- [x] signing to web3signer
- [x] acceptance-tests to web3signer
- [ ] bls-keystore to teku

### PR DESCRIPTION
**Pull request by jframe**
## PR Description
Add code from the signers library to web3signer. This adds the keystore and the secp signing module. The bls-keystore is still being used from signers for now.

## Fixed Issue(s)

fixes #771 

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

**Review by siladu — COMMENTED** [review]
Overall looks pretty straight forward but will leave Uzi to approve.

IIUC you've added a new module `keystorage` and put the `secp` module under `signing`. Can you explain the design/reasoning for this arrangement please?

Also, where you've removed TOML support, it wasn't clear to me what the equivalent/replacement of this functionality is - is it already handled by the existing YAML functionality?

nit: all the new files have old copyrights, should be easy enough to bulk update but don't know how much it matters.

**Comment by jframe** [comment]
> Overall looks pretty straight forward but will leave Uzi to approve.
> 
> IIUC you've added a new module `keystorage` and put the `secp` module under `signing`. Can you explain the design/reasoning for this arrangement please?
> 
> Also, where you've removed TOML support, it wasn't clear to me what the equivalent/replacement of this functionality is - is it already handled by the existing YAML functionality?
> 
> nit: all the new files have old copyrights, should be easy enough to bulk update but don't know how much it matters.

Fundamentally the secp module is part of the signing code. And basically, there is no benefit in making the secp256k1 a module under the signing. Unlike the signers library, there is no need to share this as a dependency with other projects so I can make this simpler by putting it into a package instead adding yet another module. Having the secp code as a module would also make it inconsistent with the approach taken for signing modules for bls and filecoin which exist as packages in the signing module rather than sub-modules.

The keystorage module on the other hand IMHO seemed separate enough that it belonged as its own module.

Web3Signer only uses YAML for configuration. The TOML functionality was only needed for EthSigner.

re: copyrights. I just copied as is, but I probably should do a bulk update should be quick to do.

File: keystorage/build.gradle [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022 ConsenSys AG.
```
[the remark this answers is not in the document]
Comment (jframe) [reply keystorage/build.gradle:2]: Will do a find/replace of the copyright dates and update them. Not sure it matters but might as well fix it up.

**Review by usmansaleem — APPROVED** [review]
LGTM

**Pull request merged**